### ISSUE TITLE
Failing workflows as github changed several compilers from ubuntu machines.

### ISSUE THREAD

**Issue by stefanofiorentino** [skypjack/uvw#270]
I'm working on a PR and it's impossible to rely on the CI as several are failing on build tools installation.
Reference: https://github.com/actions/runner-images/issues/3235

**Comment by skypjack** [comment]
Oh, yeah, I had the same problem in another project of mine, `EnTT`. It's due to a recent update on the GH side.
It's fixed in the other repo and we can freely copy the workflow to make it work properly also on `uwv`. 👍 
I can do it on Monday or tomorrow if I find enough free time.

**Comment by stefanofiorentino** [comment]
Ok, let's review this on monday than.
Currently that solution is not working to me, though.
Have a nice weekend.

**Comment by stefanofiorentino** [comment]
Here is the ubuntu-latest preinstalled packages: https://github.com/actions/runner-images/pull/6881

**Comment by skypjack** [comment]
Yeah, I think the main problem with only using the most recent compilers is that we don't check `uvw` on slightly older versions, the same that are likely in use on the target platforms instead.

**Comment by stefanofiorentino** [comment]
As 20.04 is still LTS, I fully agree with you 👍

**Comment by stefanofiorentino** [comment]
As I like more additive processes (diffusion or 3d printing) than subtractive ones (milling), 
that's my try to cleanup the workflow folder. Let me know, otherwise the EnTT approach is 100% valid to me, too.
Sorry for the high level of noise over this "side"-topic.
